packages/logger/README.md

```diff
@@ -0,0 +1,4 @@
+# @loopback/logger
+
+A LoopBack component for logging.
+
```

Comment:
This will obviously need more content, see https://loopback.io/doc/en/contrib/README-guidelines.html

Let's leave that for the end though, we need to work out the design and implementation first.